test_run.sh

```diff
@@ -208,6 +208,10 @@ test_scala_junit_test_can_fail() {
   action_should_fail test test_expect_failure/scala_junit_test:failing_test
 }
 
+test_scalaopts_from_scala_toolchain() {
```

Comment:
we aren't calling this are we?

Also, I *thought* the point of using the bazel testing framework was to stop adding things to this giant script. Can we not express this test as something that we can call with `bazel test `?

Reply:
True. I must've missed it when cleaning the test in favor to the integration test framework